GameData/MOARdV/MAS_ASET/FlagIndicator/MAS_FI_Commnet.cfg

```diff
@@ -0,0 +1,80 @@
+﻿PROP
+{
+	name = MAS_FI_Commnet

```

Comment:
Flag indicators are configured through propFlagIndicator.xml - an entry needs to be added there to ensure that this prop stays updated correctly.

Reply:
done